cerberus-web/src/main/java/com/nike/cerberus/jwt/JwtSecretData.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2021 Nike, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nike.cerberus.jwt;
+
+import java.util.LinkedList;
+
+/** A POJO that represents the JWT config */
+public class JwtSecretData {
+  private LinkedList<JwtSecret> jwtSecrets = new LinkedList<>();
```

Comment:
I believe you can just add @Getter and @Setter to this line and delete the rest of the file